posts/2025-03-11-outreachy-interns-experience/index.qmd

```diff
@@ -0,0 +1,162 @@
+---
+title: "Outreachy Interns with Bioconductor"
```

Comment:
I recommend adding the cohort to make the title more specific, such as `Outreachy December 2024 Interns with Bioconductor`